spec/acceptance/agent_spec.rb

```diff
@@ -4,15 +4,22 @@
   if version == '2.4' && default[:platform] =~ %r{(ubuntu-16.04-amd64|debian-9-amd64)}
     next
   end
+  # Zabbix supports only Debian 10 on Zabbix 4
+  if default[:platform] =~ %r{debian-10-amd64}
+    next unless version == '4.0'
```

Comment:
Does this need tweaking now that https://github.com/voxpupuli/puppet-zabbix/pull/664 has been merged?